micro/reify_test.go

```diff
@@ -14,7 +13,7 @@ scheme code:
 	(let
 		(
 			(a1 `(,x . (,u ,w ,y ,z ((ice) ,z))))
-			(a2 `(,y . corn))
+			(a2 `(,y . (corn)))
```

Comment:
It might have been a typo or it could be an improper list on purpose.  Are you sure about this?

Reply:
I was lazy. Fixed

Reply:
Seems the test is fixed :) thanks you.  Now just the comment.